nexus/src/app/sagas/region_replacement_start.rs

```diff
@@ -554,7 +555,30 @@ async fn srrs_replace_region_in_volume(
         .await
         .map_err(ActionError::action_failed)?;
 
-    Ok(())
+    info!(log, "replacement returned {:?}", volume_replace_region_result);
```

Comment:
i wonder whether this should be info or debug?

it might also be nice to include the volume ID, region ID, and stuff here, unless that's in the saga context?

Reply:
changed to debug in d2135f21b

yeah, all that context should be in the saga context